core/lib/tasks/migrations/delete_prices_with_nul_amount.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+namespace :solidus do
+  namespace :migrations do
+    namespace :delete_prices_with_nul_amount do
```

Comment:
Very small typo, here it should be either `null` (one `l` is missing) or `nil`. For consistency with the rest of the occurrences in this file, I'd personally go with `nil`. Also, the filename should change accordingly.